decidim-core/app/cells/decidim/author/profile_inline.erb

```diff
@@ -1,5 +1,5 @@
 <span class="author__avatar">
-  <%= image_tag model.avatar_url %>
+  <%= image_tag model.avatar_url, alt: "Avatar" %>
```

Comment:
Shouldn't we translate this?

Reply:
Don't worry about the translations, you just need to add it in English and the rest will be managed via Crowdin! But we do need to change it to `<%= t('.avatar') %>` (or similar) and add it to `config/locales/en.yml`